homework/index.js

```diff
@@ -0,0 +1,24 @@
+const inputValue = document.querySelector('#search');
+const searchButton = document.querySelector('.searchButton');
+const reposContainer = document.querySelector('.main__profile-repos');
+// const clientId = '1ebd35cdc9277ed1ee8c';
+// const clientSecret = '0ecdc67b56e4ca7c3b4710b60f135d52a8f11a32';
+const fetchRepo = async () => {
+  // const apiCall = await fetch(`https://api.github.com/users/${user}?clientId=${clientId}&clientSecret=${clientSecret}`);
+  const apiCall = await fetch('https://api.github.com/HackYourFuture/repos?callback=?');
+
+  const data = await apiCall.json();
+  return {
+    data,
```

Comment:
Here you're 'wrapping' the `data` in an object. There's no need;  you can just return `data` without putting it in a wrapper object. Then you also don't need to store it in a variable - you can just return it directly if you prefer.
(+1 for use of shorthand properties though!)